Sources/StreamChatUI/MessageActionsPopup/ChatMessageActionsView.swift

```diff
@@ -5,27 +5,32 @@
 import StreamChat
 import UIKit
 
+/// View that contains `_ChatMessageActionItem`s.
+/// It is shown in `_ChatMessagePopupVC` to show the defined action items.
 public typealias ChatMessageActionsView = _ChatMessageActionsView<NoExtraData>
 
+/// View that contains `_ChatMessageActionItem`s.
+/// It is shown in `_ChatMessagePopupVC` to show the defined action items.
 open class _ChatMessageActionsView<ExtraData: ExtraDataTypes>: _View, UIConfigProvider {
```

Comment:
TBH this view feels completely redundant and just complicates everything. There's nothing happening here really, and the main thing (putting the buttons up there) should be IMO done by the VC.

Reply:
true, I deleted it and used `ContainerView` directly 👍 good catch